doc/03-component-interfaces/03-component-interfaces.md

```diff
@@ -132,7 +132,10 @@ hierarchy diagram using whatever tools you would like. Then, include a picture
 of it in this folder. You may also embed it just below using markdown syntax
 (i.e., `![ALT TEXT](path/to/file)`).
 
-<!-- TODO: make a diagram of your component hierarchy then delete this comment -->
+`![Music playlist inventory](doc/03-component-interfaces/Untitled Notebook (5).pdf)`
```

Comment:
I am not able to see this because it was not committed. PDFs are ignored and cannot be embedded in markdown. Make it a jpg or png.